### PR TITLE
Replace usages of TestStream with GENERATE_STREAM

### PR DESCRIPTION
**Pull request by gierlachg**
Replaced usages of `TestStream` with `GENERATE_STREAM` - I will be changing `TestStream` to be able to support custom data, similarly to `TestBatch`.

### CODE REVIEW

**Review by hakanmemisoglu — APPROVED** [review]

File: hazelcast-sql/src/test/java/com/hazelcast/jet/sql/impl/schema/SqlMappingTest.java [hunk 1]
```diff
@@ -165,10 +166,10 @@ private void test_alias(String javaClassName, String... aliases) {
 
     @Test
     public void test_caseInsensitiveType() {
-        sqlService.execute("CREATE MAPPING t1 TYPE TestStream");
-        sqlService.execute("CREATE MAPPING t2 TYPE teststream");
-        sqlService.execute("CREATE MAPPING t3 TYPE TESTSTREAM");
-        sqlService.execute("CREATE MAPPING t4 TYPE tEsTsTrEaM");
+        TestBatchSqlConnector.create(sqlService, "TestBatch", 1);
```
Comment (viliam-durina) [inline hazelcast-sql/src/test/java/com/hazelcast/jet/sql/impl/schema/SqlMappingTest.java:169]: This is case-insensitive mapping name, but the test was about case-insensitive mapping type.

**Review by viliam-durina — CHANGES_REQUESTED** [review]

**Comment by gierlachg** [comment]
run-lab-run

**Comment by viliam-durina** [comment]
run-lab-run

**Comment by gierlachg** [comment]
run-lab-run

**Comment by gierlachg** [comment]
run-lab-run

**Pull request merged**